config/locales/en/main.yml

```diff
@@ -857,7 +857,9 @@ en:
               blank_unless_goto_end: "This field is required unless you choose 'skip to end of form'."
         user:
           at_least_digits: "Please enter at least %{num} digits."
-          cant_delete_if_responses: "You can't delete this user because they have associated responses."
+          cant_delete_if_has_responses: "You can't delete this user because they have associated responses."
+          cant_delete_if_has_reviewed_responses: "You can't delete this user because they have reviewed responses."
+          cant_delete_if_has_sms_messages: "You can't delete this user because they have associated SMS messages."
```

Comment:
Will we know to add a new string here if we add a new type of user association? Maybe we could at least note it in the new `destruction.md`...

Reply:
Alternatively could we call it `cant_delete_if_has_associated` and end with `they have an associated: %{type}` so that we don't need to do dynamic key lookup?

Reply:
The issue is then we need to translate the `type` parameter, which becomes a whole ordeal and we then have to remember to do that. I actually tried it out after reading your suggestion but it gets messy fast.

Generally passing meaty params to i18n like this is almost always problematic due to pluralization rules or things like this. 

I did document it in the .md file though. That's a good idea. Thanks!